infra/aws/e2e/accounts/tf/accounts.tf

```diff
@@ -0,0 +1,182 @@
+terraform {
+  required_version = "~> 0.13"
```

Comment:
All of our other terraform modules require a minimum of 0.12 at present; can this be downgraded to 0.12?